src/components/panels/BountyPanel.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <HubPanelWrap :title="headertext">
+  <HubPanelWrap :title="headertext" class="bounties" :class="[syndicate]">
```

Comment:
not really a fan of how much this explodes the class list

![image](https://user-images.githubusercontent.com/7128721/63781519-7b00e900-c8af-11e9-8da1-88274a0f273d.png)

```suggestion
  <HubPanelWrap :title="headertext" class="bounties">
```

Reply:
Oops, that should have been something like `:class="[type.toLowerCase()]"`. But looks like I ended up using only `.bounties`, so the syndicate part can be removed completely.